.circleci/config.yml

```diff
@@ -32,38 +29,84 @@ commands:
       - run:
           name: docs
           command: make build-docs
+  install_dep_torch_151:
+    steps:
+      - checkout
+      - run:
+          name: install dependencies with torch 1.5.1
+          command: |
+            echo "\$conda_bin install -y -q pytorch==1.5.1 -c pytorch" >> ./setup_dev_env.sh
```

Comment:
this will install twice torch since when environment is created already installs torch. The better is to set an environnment variable PYTORCH_VERSIO